pkg/jobqueue/dbjobqueue/schemas/005_jobs_expiry.sql

```diff
@@ -0,0 +1,19 @@
+-- add the expires_at column
+ALTER TABLE jobs
+ADD COLUMN expires_at timestamp DEFAULT NOW() + interval '14 days';
+
+-- set the expires at to a sane value
+UPDATE jobs SET expires_at = queued_at + interval '14 days';
```

Comment:
Does the DEFAULT prefill the column? Making the UPDATE redundant?

Reply:
The update issue is a good one, that might take up a lot of space that isn't necessarily there. I've removed the UPDATE statement for now, we can always run it manually.

As far as the DEFAULT goes, it would insert the timestamp of when the schema change is executed. This isn't *that* big of a deal, we can fix that as with the above :)